src/Tokenizer/Analyzer/Analysis/AttributeAnalysis.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer\Analysis;
+
+/**
+ * @internal
+ */
+final class AttributeAnalysis
```

Comment:
Conflicts with work done in #7395 which is ongoing for a while and took a lot of effort before, I really would not like to discard it in favour of this PR. Maybe we can extract that work from the `OrderedAttributesFixer` PR, review it as a standalone feature, and then rebase both PRs on that analyzer?

Reply:
@HypeMC what do you think?

Reply:
@Wirone Sure, I'll extract the code to a separate PR. I think that'll make reviewing it easier.

Reply:
@Wirone See #7909